src/entity/Vacancy.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm"
```

Comment:
! to all files, fix: 
1. add prettier and format all files
2. add a new line at the end of the file where not existed
3. remove unnecessary console.log \ comments and new lines